docs/data/ampli/migration.md

```diff
@@ -178,12 +178,14 @@ amplitude.init(AMPLITUDE_API_KEY);
 ```typescript
 import { ampli, SongPlayed } from './ampli';
 
-// Initialize with an Environment from Data
+// Initialize with Amplitude API Key
 ampli.load({
-  environment: 'my-production'
```

Comment:
Is there a reason we're leaving `environment` in the some snippets while removing it in others?

Reply:
re-checked together, seems everything is updated now